cmd/telemeter-server/main.go

```diff
@@ -325,7 +322,12 @@ func (o *Options) Run() error {
 		})
 	}
 	{
-		external := http.NewServeMux()
+		external := chi.NewRouter()
```

Comment:
nit: above we call the router `r` and the mux `internal` and here we are following a different pattern. let's keep it consistent.

Reply:
The problem is that the above still needs the internal to begin with. We need to refactor `telemeter_http.DebugRoutes` etc in another PR and then can have consistency.

Reply:
no I simply mean we should call the router `r` in both cases